_content/articles/pure-land-sects-of-buddhism_blacker-carmen.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "The Pure Land Sects of Buddhism"
+authors:
+  - "Carmen Blacker"
+external_url: "https://www.nembutsu.info/blacker.htm"
+drive_links:
+  - "https://drive.google.com/file/d/1ZLtPaqeLwJ0e5L1WqU-J-vv-ELI1SaaG/view?usp=sharing"
+course: japanese
+tags:
+  - pureland
+  - buddha
+journal: "MURYOKO 'Infinite Light': Journal of Shin Buddhism"
```

Comment:
Just "Journal of Shin Buddhism" is okay.  If you can't find a volume for this, go ahead and move this field to publisher and move this file to essays.